mirrord/agent/iptables/src/lib.rs

```diff
@@ -212,6 +218,11 @@ where
             redirect = Redirects::FlushConnections(FlushConnections::create(Box::new(redirect))?)
         }
 
+        if with_mesh_exclusion {
+            redirect =
+                Redirects::WithMeshExclusion(WithMeshExclusion::create(ipt, Box::new(redirect))?)
+        }
+
```

Comment:
Maybe worth leaving a comment that this order must not change, i.e. `WithMeshExclusion` stays in the outer layer, or else `Redirect::exclusion` will return `None`. WDYT? Same with `Redirect::load`.